pages/contribute/github_way.md

```diff
@@ -13,7 +13,14 @@ permalink: github_way.html
 ## Make a GitHub account
 1. Make a [GitHub account](https://docs.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account).
 
-## Announce and discuss your proposal through GitHub "issues"
+## Contribution process
+The process of contribution via GitHub is sketched below. 
+
+{% include image.html file="github_way_flow.svg" alt="Process of contributing via GitHub" max-width="1500"  %}
```

Comment:
```suggestion
{% include image.html file="github_way_flow.svg" alt="Process of contributing via GitHub" max-width="845px"  %}
```
If you go over 845pc the image will get out of the body of the page, causing a scroll bar. I saw that the image is quite wide and with small letters which is not ideal in this case.

Reply:
Hi @bedroesb  the boxes in @floradanna 's google doc did not match the text headings. If you could export her slide as sag please go ahead.

Reply:
Hi, yes indeed the text in the flowchart doesn't correspond to the paragraphs yet. However, we could keep the text in the image very short, so that we can increase the font size and make the text easily readable. The same short sentence could be the title of a paragraphs, but then we should add a brief text between the title and the bullet points to summarise what that step is about, I think. Anyway, as Pinar said, all the info for testing are there and we can do the rest later.